main.pm

```diff
@@ -562,6 +566,20 @@ sub load_x11tests(){
     loadtest "x11/shutdown.pm";
 }
 
+sub load_autoyast_tests(){
+#    init boot in load_boot_tests
+    loadtest("autoyast/system.pm");
+    loadtest("autoyast/console.pm");
+    loadtest("autoyast/login.pm");
+    loadtest("autoyast/repos.pm") unless get_var("SUPPORT_SERVER_GENERATOR");
+    loadtest("autoyast/autoyast_verify.pm") if get_var("AUTOYAST_VERIFY");
+    loadtest("autoyast/useradd.pm") unless get_var("INSTALLONLY");
```

Comment:
Why do we need a test to add a user? why can't we have autoyast templates that do that properly?

Users aren't going to have magical openQA to add a user after their installation using autoyast, so we really need to be testing the addition of users VIA autoyast..not scripting around it with openQA tests